tests/integration/test_check_config.py

```diff
@@ -68,30 +69,21 @@ async def test_good_config_validates_successfully(
 @pytest.mark.abort_on_fail
 async def test_bad_config_sets_action_results(ops_test, prometheus_charm, prometheus_tester_charm):
     """Deploy Prometheus and a single client with a good configuration."""
-    await asyncio.gather(
-        ops_test.model.deploy(
-            "prometheus-scrape-config-k8s",
-            channel="edge",
-            application_name=scrape_shim,
-            config={"scrape_interval": "NotANumber!!!"},
-        ),
-        ops_test.model.deploy(
-            prometheus_tester_charm,
-            resources=scrape_tester_resources,
-            application_name=bad_scrape_tester,
-        ),
-    )
-    await ops_test.model.wait_for_idle(apps=[scrape_shim, bad_scrape_tester])
-
-    await asyncio.gather(
-        ops_test.model.add_relation(
-            f"{bad_scrape_tester}:metrics-endpoint", f"{scrape_shim}:configurable-scrape-jobs"
-        ),
-        ops_test.model.add_relation(
-            f"{prometheus_app_name}:metrics-endpoint", f"{scrape_shim}:metrics-endpoint"
-        ),
+    await ops_test.model.wait_for_idle(apps=[prometheus_app_name])
```

Comment:
is it being deployed in a different test (file)?
Are we not doing 'one model per test file/module' anymore? We should.

Reply:
It's being deployed as a fixture I believe. Still one model per file.